code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -20,13 +20,13 @@
 
 
 //These vars are related to how mobs locate and target
-	var/robust_searching = 0 //By default, mobs have a simple searching method, set this to 1 for the more scrutinous searching (stat_attack, stat_exclusive, etc), should be disabled on most mobs
+	var/robust_searching = 0 //Set this to 0 for simple targetting, Set this to 1 for the more scrutinous searching (stat_attack, stat_exclusive, etc)
 	var/vision_range = 9 //How big of an area to search for targets in, a vision of 9 attempts to find targets as soon as they walk into screen view
 	var/aggro_vision_range = 9 //If a mob is aggro, we search in this radius. Defaults to 9 to keep in line with original simple mob aggro radius
 	var/idle_vision_range = 9 //If a mob is just idling around, it's vision range is limited to this. Defaults to 9 to keep in line with original simple mob aggro radius
 	var/search_objects = 0 //If we want to consider objects when searching around, set this to 1. If you want to search for objects while also ignoring mobs until hurt, set it to 2. To completely ignore mobs, even when attacked, set it to 3
 	var/list/wanted_objects = list() //A list of objects that will be checked against to attack, should we have search_objects enabled
-	var/stat_attack = 0 //Mobs with stat_attack to 1 will attempt to attack things that are unconscious, Mobs with stat_attack set to 2 will attempt to attack the dead.
+	var/stat_attack = 1 //Mobs with stat_attack to 1 will attempt to attack things that are unconscious, Mobs with stat_attack set to 2 will attempt to attack the dead.

```

Comment:
Does this include people in *critical* health status? If so, this will cause more problems than it solves. The chances of rescuing people from a hostile mob attack (ignoring lone miners, as if they die, they are screwed the rest of the round) will become extremely low. There needs to be a differentiation between *unconscious* and *critical*.